spec/ndx-microscopy.extensions.yaml

```diff
@@ -95,7 +95,7 @@ groups:
       doc: General estimate of location in the brain being subset by this space.
         Specify the area, layer, etc.
         Use standard atlas names for anatomical regions when possible.
-        Specify 'whole-brain' if the entire brain is strictly contained within the space.
+        Specify 'whole brain' if the entire brain is strictly contained within the space.
```

Comment:
Also sneaking this change in

Reply:
From @alessandratrapani, has not seen the version with the dash, but intuitively wanted 'Whole Brain'